manifests/init.pp

```diff
@@ -31,6 +31,7 @@
   },
   $createkeys_path     = '/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin',
   $manage_eyaml_config = true,
+  $manage_keys         = true,
```

Comment:
We do enforce it with the code above. This is done because of Puppet's history of handling boolean values poorly. It aligns with the principal of least surprise, so if someone does pass a stringified boolean, things still work as expected.